docs/README.md

```diff
@@ -0,0 +1,351 @@
+# Test and config files
+
+A typical `*-bench` project runs checks specified in `controls` files that are a YAML 
+representation of the CIS Benchmark checks. There is a 
+`controls` file per version and node type.
+
+`controls` for the various versions can be found in directories
+with same name as the CIS versions under `cfg/`, for example `cfg/cis-1.4.0`.
+`controls` are also organized by distribution under the `cfg` directory, for
+example `cfg/rh-0.7`.
+
+## Controls
+
+`controls` is a YAML document that contains checks that must be run against a 
+specific Kubernetes node type, master or node and version.
+
```

Comment:
Again Kubernetes specific. Maybe better: "`controls` is a YAML document containing the test definitions for a benchmark"